src/include/utils/unsync_guc_name.h

```diff
@@ -144,6 +144,7 @@
 		"enable_partition_pruning",
 		"enable_partitionwise_aggregate",
 		"enable_partitionwise_join",
+		"enable_password_profile",
```

Comment:
why in unsync guc list ?

Reply:
Here, as we always login in database from master node, only need to set this guc in master.

Reply:
> Here, as we always login in database from master node, only need to set this guc in master.

If don't sync, does it impact login in with profile on segments?

Reply:
yes, no impact, as we process login logic only in master, QE will fork in buildgang.

Reply:
In my opinion, unsync or sync are both OK, unsync maybe is better.